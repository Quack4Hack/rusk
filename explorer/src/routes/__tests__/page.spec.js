import { describe, expect, it } from "vitest";
import { render } from "@testing-library/svelte";
import ChainInfo from "../+page.svelte";

describe("Chain Info", () => {
  it("should render the Chain Info page", () => {
    const { container } = render(ChainInfo, {});

    expect(container.firstChild).toMatchSnapshot();
  });
});
