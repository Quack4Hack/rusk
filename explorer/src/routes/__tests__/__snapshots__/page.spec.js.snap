// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Chain Info > should render the Chain Info page 1`] = `
<div>
  <div>
    Chain Info
  </div>
  <!--&lt;+page&gt;-->
</div>
`;
